releasenotes/notes/higher-order-spsa-f2eddb44b00b82e0.yaml

```diff
@@ -0,0 +1,15 @@
+---
+features:
+  - |
+    Add the :class:`~qiskit.algorithms.optimizers.QNSPSA` optimizer routine to the optimizers,
+    and support second-order SPSA in :class:`~qiskit.algorithms.optimizers.SPSA`.
```

Comment:
I wonder if this would make more sense being split into two release notes, one for QNSPSA and the other for 2-SPSA. I also would probably put a module reference to these things living in `qiskit.algorithms` just for sign posting if people don't click the links. But that's something I'll clean up at release time when I prepare the release notes for release. Right now this is good enough (and helps me out a lot just having the references here already).